README.md

```diff
@@ -63,11 +63,11 @@ Distributed under the XYZ license. See ``LICENSE`` for more information.
 
 ## Contributing
 
-1. Fork it (<https://github.com/yourname/yourproject/fork>)
-2. Create your feature branch (`git checkout -b feature/fooBar`)
-3. Commit your changes (`git commit -am 'Add some fooBar'`)
-4. Push to the branch (`git push origin feature/fooBar`)
-5. Create a new Pull Request
+1. Fork it. (<https://github.com/yourname/yourproject/fork>)
+2. Create your feature branch. (`git checkout -b feature/fooBar`)
+3. Commit your changes. (`git commit -am 'Add some fooBar'`)
+4. Push to the branch. (`git push origin feature/fooBar`)
```

Comment:
Why do you put full stop *before* parenthesis?

Reply:
Yes, if you want to dot those lines it should really be:

> Text text (`…`).

…or perhaps even better - commas, and a period on the last.

> 1. Foo,
> 2. Bar,
> 3. Baz.

/2¢

Reply:
Parentheses are unneeded if it it split over lines. Keep the full stop.

Example code:

```markdown
2. Create your feature branch.
    ```sh
    git checkout -b feature/fooBar
    ```
```

Result:

2. Create your feature branch.
    ```sh
    git checkout -b feature/fooBar
    ```